presto-main/src/main/java/io/prestosql/event/QueryMonitor.java

```diff
@@ -150,6 +150,7 @@ public void queryImmediateFailureEvent(BasicQueryInfo queryInfo, ExecutionFailur
                         ofMillis(0),
```

Comment:
The commit message is too long. We can shorten it while conveying the essence of the change: "Add resource waiting time to query completion event"

Reply:
Done